libs/SalesforceAnalytics/SalesforceAnalytics/Classes/Logger/SFSDKLogger.h

```diff
@@ -144,4 +197,103 @@
  */
 - (void)log:(nonnull Class)class level:(DDLogLevel)level format:(nonnull NSString *)format, ...;
 
+/**
```

Comment:
I contemplated this when I was initially looking at solutions.  In a more classic OO design pattern, it seems that you'd separate your `LoggingConfiguration` class (which would be more of an instance class) from your `Logger` class (which would be more of a service class that owned `LoggingConfiguration` instances).  In the current design, those roles are both aggregated in the same class.

While I think I'd like the clean separation in that approach, it seemed like a bigger departure from the current implementation.  It's an idea I could poke at, though, if you guys think that would be better.